schema.graphql

```diff
@@ -14,7 +14,6 @@ type Token @entity {
   symbol: String!
   name: String!
   decimals: BigInt!
-  totalSupply: BigInt!
```

Comment:
we are not going to use it for anything?

Reply:
Not for the moment. It is more complicated to implement than what we are gaining (currently nothing) as some tokens have a dynamic total supply (i.e. Honey)